ToDoApp/ToDoApp.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -1,76 +0,0 @@
-{
```

Comment:
I didn't want you to remove SwiftLint package at all. I just wanted you to remove it from the build phase. The reason for this is that I would like you to add it back to build phases before you push new changes and verify there are no formatting issues by using SwiftLint. I agree it is not convenient to add/remove it, but this is yet unstable, unfortunately :( 
So I want you to run swift lint for this branch as well and fix the formatting issues